pkg/metrics/metrics.go

```diff
@@ -98,13 +98,13 @@ The label 'result' can have the following values:
 
 	// Metrics tied to the queue system.
 
-	GitVersion = prometheus.NewGaugeVec(
+	gitVersion = prometheus.NewGaugeVec(
```

Comment:
```suggestion
	buildInfo = prometheus.NewGaugeVec(
```
but analogy to the analogous metric in k8s core